app/javascript/dashboard/routes/dashboard/settings/reports/components/CsatTable.vue

```diff
@@ -0,0 +1,191 @@
+      csatResponses: 'csat/getCSATResponses',
+      metrics: 'csat/getMetrics',
+    }),
+    columns() {
+      return [
+        {
+          field: 'contact',
+          key: 'contact',
+          title: this.$t('CSAT_REPORTS.TABLE.HEADER.CONTACT_NAME'),
+          align: 'left',
+          width: 200,
+          renderBodyCell: ({ row }) => {
+            if (row.contact) {
+              return <UserAvatarWithName size="24px" user={row.contact} />;
+            }
+            return '---';
+          },
+        },
+        {
+          field: 'assignedAgent',
+          key: 'assignedAgent',
+          title: this.$t('CSAT_REPORTS.TABLE.HEADER.AGENT_NAME'),
+          align: 'left',
+          width: 200,
+          renderBodyCell: ({ row }) => {
+            if (row.assignedAgent) {
+              return (
+                <UserAvatarWithName size="24px" user={row.assignedAgent} />
+              );
+            }
+            return '---';
+          },
+        },
+        {
+          field: 'rating',
+          key: 'rating',
+          title: this.$t('CSAT_REPORTS.TABLE.HEADER.RATING'),
+          align: 'center',
+          width: 80,
+          renderBodyCell: ({ row }) => {
+            const [ratingObject = {}] = CSAT_RATINGS.filter(
+              rating => rating.value === row.rating
+            );
+            return (
+              <span class="emoji-response">{ratingObject.emoji || '---'}</span>
+            );
+          },
+        },
+        {
+          field: 'feedbackText',
+          key: 'feedbackText',
+          title: this.$t('CSAT_REPORTS.TABLE.HEADER.FEEDBACK_TEXT'),
+          align: 'left',
+        },
+        {
+          field: 'converstionId',
+          key: 'converstionId',
+          title: '',
+          align: 'left',
+          width: 100,
+          renderBodyCell: ({ row }) => {
+            const routerParams = {
+              name: 'inbox_conversation',
+              params: { conversation_id: row.conversationId },
+            };
+            return (
+              <router-link to={routerParams}>
+                {`#${row.conversationId}`}
+              </router-link>
+            );
+          },
+        },
+      ];
+    },
+    tableData() {
+      return this.csatResponses.map(response => ({
+        contact: response.contact,
+        assignedAgent: response.assigned_agent,
+        rating: response.rating,
+        feedbackText: response.feedback_message || '---',
+        conversationId: response.conversation_id,
+      }));
+    },
+  },
+  methods: {
+    onPageNumberChange(pageIndex) {
+      this.$emit('page-change', pageIndex);
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+.csat--table-container {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+
+  .ve-table {
+    background: var(--white);
+
+    &::v-deep {
+      .ve-table-container {
+        border-radius: var(--border-radius-normal);
+        border: 1px solid var(--color-border) !important;
```

Comment:
Is it possible to avoid this important with  nested selector or `>`?

Reply:
Tried that! It doesn't work.